packages/x-scheduler/src/joy/event-calendar/EventCalendar.css

```diff
@@ -1,9 +1,48 @@
-.EventCalendar {
-  width: 300px;
-  height: 300px;
+.EventCalendarRoot {
   box-sizing: border-box;
-  border: 1px solid red;
+
+  width: 814px;
+  height: 600px;
+  border: 1px solid var(--gray-4);
```

Comment:
@noraleonte for those variables, not sure if you think we should focus on dark mode for now or just all turn on the light mode and do a dark mode later on.